.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```

Comment:
Can you add a prefix to title? Something like `[Bug]: ` and `[Feature Request]: ?